sdk-extensions/autoconfigure/README.md

```diff
@@ -191,6 +191,12 @@ These properties can be used to control the maximum size of recordings per span.
 | otel.span.event.count.limit            | OTEL_SPAN_EVENT_COUNT_LIMIT            | The maximum number of events per span. Default is `128`.               |
 | otel.span.link.count.limit             | OTEL_SPAN_LINK_COUNT_LIMIT             | The maximum number of links per span. Default is `128`                 |
 
+## Exemplars
+
+| System property          | Environment variable     | Description                                                                       |
+|--------------------------|--------------------------|-----------------------------------------------------------------------------------|
+| otel.metrics.exemplar.filter | OTEL_METRICS_EXEMPLAR_FILTER | The filter for exemplar sampling.  Can be `NONE`, `ALL` or `WITH_SAMPLED_TRACE`. Default is `WITH_SAMPLED_TRACE`.|
```

Comment:
Not for this PR since we're correctly reflecting the spec, but all of our other enum-type SDK config uses lowercase, not uppercase, would be nice if it can be fixed @jsuereth I don't know if we have time before the release though

Reply:
This is a reasonable thing to fix.  If all other spec uses lower case, that's an easy bugfix.  We're only in feature-freeze right now.

Reply:
If it's ok, I'll fix this in a follow up PR once [this](https://github.com/open-telemetry/opentelemetry-specification/pull/1965) is submitted.